x/programs/cmd/simulator/cmd/program.go

```diff
@@ -49,7 +49,7 @@ func (c *programCreateCmd) Run(ctx context.Context, log logging.Logger, db *stat
 		return newResponse(0), err
 	}
 	if !exists {
-		return newResponse(0), fmt.Errorf("%w: %s", ErrNamedKeyNotFound, c.keyName)
+		return newResponse(0), fmt.Errorf("%w: %s", ErrNamedKeyNotFound, *c.keyName)
```

Comment:
@najeal do you know why this is not caught at compile-time ? Isn't Go statically typed ?

Reply:
Yes Go is statically typed. I think validating the type over a string verb (ex: "%s") is out of scope of the compiler.
It is caught by `go vet` tool when running `go test`. The tool is enabled by default when using `go test` (you can disable it with `-vet=off`).

Reply:
Thanks!

Reply:
Yep, again an issue with CI